src/style.css

```diff
@@ -0,0 +1,825 @@
+@import url("https://fonts.googleapis.com/css2?family=Lato&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Lato:wght@900&display=swap");
+@import url("http://fonts.cdnfonts.com/css/cocogoose");
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "lato", Arial, Helvetica, sans-serif;
+  color: rgba(39, 42, 49, 0.8);
+}
+
+.dn {
+  display: none;
+}
+
+.dn-feature {
+  display: none !important;
+}
+
+li {
+  list-style: none;
+}
+
+a {
+  text-decoration: none;
+}
+
+/* navbar and headline styles */
+.display-none {
+  display: none;
+}
+
+/* Style for mobile menu */
+.mobile-menu {
+  position: fixed;
+  z-index: 1;
+  background-color: #323232;
+  height: 100vh;
+  width: 100vw;
+  opacity: 0.96;
+}
+
+.mobile-nav {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 40px;
+  margin: 30px;
+  margin-top: 30px;
+}
+
+.mobile-nav li {
+  border-bottom: 2px solid #ec5242;
+  padding-bottom: 25px;
+  width: 40%;
+  text-align: center;
+}
+
+.headline__contact--email {
+  text-align: center;
+  margin-bottom: 30px;
+}
+
+.headline__contact--email > a {
+  border-bottom: 2px solid #272a31;
+  color: #272a31;
+}
+
+.mobile-nav li a {
+  color: #d3d3d3;
+  font-size: 40px;
+}
+
+.partners__list li {
+  width: 30vw;
+  padding-bottom: 15px;
+}
+
+.mobile-nav li:first-child {
+  border-bottom: none;
+  padding-bottom: 0;
+}
+
+.x-icon {
+  font-size: 30px;
+  border: none;
+  color: #d3d3d3;
+  text-align: right;
+  justify-self: flex-end;
+  margin-left: 300px;
```

Comment:
- [ ] Please try to modify the positioning of your close button for the mobile version menu. Right now, if your phone screen is smaller than 460px width, you cannot see the button properly, and if you hit 435px width you can no longer see the button, which makes it impossible to close this menu. You can solve this issue by using `rem` or `em` units instead of pixels or if you find any other solution to this issue it's also good!